[Bug] generate-velocity-report.sh: 集計期間の日付パースが失敗する

## 概要

`generate-velocity-report.sh` でベロシティ集計期間の開始日・終了日のパースに失敗し、ジョブがエラー終了する。

## 再現環境

- ワークフロー: `⑤ 統合プロジェクト分析`
- リポジトリ: `lurest-inc/github-projects-starter-kit`
- 実行ログ: https://github.com/lurest-inc/github-projects-starter-kit/actions/runs/23200412054

## エラーメッセージ

```
  集計期間: "2026-01-26\t2026-03-22" 〜 （8 週間）
  Done アイテム数（期間内）: 0 件
jq: error (at <unknown>): date ""2026-01-26\t2026-03-22"" does not match format "%Y-%m-%d"
##[error]Process completed with exit code 5.
```

## 原因

`scripts/generate-velocity-report.sh` 131〜143行目で、jq の `@tsv` 出力を `read` で分割する際に問題が発生している。

```bash
read -r PERIOD_START PERIOD_END < <(jq -n \
  ...
  [(... | strftime("%Y-%m-%d")),
   (... | strftime("%Y-%m-%d"))] | @tsv
' | tr '\t' ' ')
```

jq の `@tsv` はタブ文字で区切った文字列を出力するが、`-r` (raw output) フラグなしで実行しているため、出力がダブルクォートで囲まれた文字列（`"2026-01-26\t2026-03-22"`）となる。この結果、`tr '\t' ' '` がリテラルなタブ文字を検出できず、`PERIOD_START` に全体が格納され、`PERIOD_END` が空になる。後続の `strptime` で不正な日付文字列としてパースエラーが発生する。

## 修正方針

jq に `-r` フラグを追加して raw output にする。

```bash
read -r PERIOD_START PERIOD_END < <(jq -rn \
  ...
)
```

または、`@tsv` の代わりに別のセパレータを使い、`read` の IFS で分割する。

```bash
IFS=$'\t' read -r PERIOD_START PERIOD_END < <(jq -rn \
  ...
  | @tsv
')
```

## 関連ファイル

- `scripts/generate-velocity-report.sh`（131〜143行目）

--- a/scripts/generate-velocity-report.sh
+++ b/scripts/generate-velocity-report.sh
@@ -128,7 +128,7 @@ EXECUTED_AT=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
 
 # 集計期間の開始日を計算（現在の ISO 週の月曜日から VELOCITY_WEEKS 週前）
 # macOS と Linux の date 互換性のため jq で計算
-read -r PERIOD_START PERIOD_END < <(jq -n \
+read -r PERIOD_START PERIOD_END < <(jq -rn \
   --argjson weeks "${VELOCITY_WEEKS}" '
   (now | floor) as $now |
   # 現在の曜日（月=0, 日=6）

